src/ontology/mondo-ingest.Makefile

```diff
@@ -21,12 +21,7 @@ pip-%:
 	 python3 -m pip install --upgrade $*
 
 .PHONY: dependencies
-dependencies:
-	$(MAKE) pip-pip
-	$(MAKE) pip-setuptools
-	$(MAKE) pip-oaklib
-	$(MAKE) pip-sssom
-	$(MAKE) pip-semsql
+dependencies: pip-pip pip-setuptools pip-oaklib pip-sssom pip-semsql
```

Comment:
I suppose this change is effectively the same and more succinct. Ah, and I see that you're saying `-B` gets passed down.